sdk/eventhub/Azure.Messaging.EventHubs/src/Compatibility/TrackOneEventHubClient.cs

```diff
@@ -163,7 +163,7 @@ internal sealed class TrackOneEventHubClient : TransportEventHubClient
         ///   and their identifiers.
```

Comment:
Orthogonal to your specific change here but do we actually plan to ship a type called `TrackOneEventHubsClient`? That doesn't seem like a great idea.

Reply:
We do, for the first preview only.  The timeline did not allow for rewriting the infrastructure needed, so we made the decision to put a shiny new set of API surface on the old track one types.   These are all destined to be killed off after the first preview in favor of a full track two implementation.

As a general rule, anything you see with `TrackOne` in the name should be in the compatibility namespace/folder for easy identification and death.